Content.Server/Paper/ActivateOnWriteComponent.cs

```diff
@@ -0,0 +1,9 @@
+namespace Content.Server.Paper;
+
+/// <summary>
+/// Activates the item when used to write on paper, as if Z was pressed.
+/// </summary>
+[RegisterComponent]
+public sealed class ActivateOnWriteComponent : Component
```

Comment:
Cant you use OnUseTimerTriggerComponent?

Reply:
i do, that doesn't handle using the explosive to write on paper

Reply:
You can just subscribe the component to InteractUsing event, maybe behind a field so no unexpected behaviors happen

Reply:
i dont think explosive triggering when written with should be in onusetimertrigger, its bloat even if its on a field

Reply:
I think my issue with this is the name, it implies that the component knows when someone starts or stops writing, change it to something less vague and it should be fine